CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+# Guidelines for upgrading or adding new extensions
+
+One of the common worklow breakages caused by new extensions, or upgraded extensions, is failures in logical restores. A few workflows to verify when making changes to extensions are essentially various combinations straddling the change:
+
+- [ ] backup/restore on a vanilla project
+- [ ] backup/restore on a project with the extension enabled
+- [ ] backup/restore on a project, after the extension has been enabled and then disabled
+- [ ] backup from an old project (w/o change), restore to a new project (w/ change)
+- [ ] backup from older project (e.g. early 13/14/15 build), restore to a new project (w/ change)
```

Comment:
should we also backup and restore with all/the majority of other extensions enabled as well as the new/updated one?

Reply:
Have we seen any such failure cases? At the limit you'll end up testing all combinations of extensions, which isn't very feasible.

Reply:
i would just suggest all.
yep, i think there were some conflicts when we initially tried adding vault

Reply:
all or just top 20, we had csv in slack with usage stats